server/chat-plugins/scavengers.ts

```diff
@@ -2205,9 +2196,9 @@ const ScavengerCommands: ChatCommands = {
 			formattedData.map(entry => {
 				const userid = toID(entry.name);
 
-				const auth = room.auth && room.auth[userid] ? room.auth[userid] :
-					Users.usergroups[userid] ? Users.usergroups[userid].charAt(0) : '&nbsp;';
-				const color = room.auth && userid in room.auth ? 'inherit' : 'gray';
+				const auth = room.auth.get(user.id) ? room.auth.get(user.id) :
+					Users.groups.get(userid).trim() ? Users.groups.get(userid) : '&nbsp;';
+				const color = room.auth.get(user.id) ? 'inherit' : 'gray';
```

Comment:
lol, I knew this was you and not sparky.

You changed this from `userid` to `user.id`, so all the table rows would be colored like the user who used the command, rather than the user whose row it was about.